### PR TITLE
Swift package manager support

### PR DESCRIPTION
**Pull request by ThasianX**

### CODE REVIEW

**Review by rpwachowski — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -8,17 +21,71 @@ build/*
...
+# It is recommended to not store the screenshots in the git repo.
+# Instead, use fastlane to re-generate the screenshots whenever they are needed.
+# For more information about the recommended setup visit:
+# https://docs.fastlane.tools/best-practices/source-control/#source-control
+
+fastlane/report.xml
+fastlane/Preview.html
+fastlane/screenshots/**/*.png
 fastlane/test_output
-fastlane/README.md
```
Comment (powerje) [inline .gitignore:24]: Not a big deal but seems unnecessary

File: Fabulous.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -391,6 +417,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.duet.fabulous.Fabulous;
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = NO;
```
Comment (powerje) [inline Fabulous.xcodeproj/project.pbxproj]: We should probably leave Catalyst support in unless we know specifically it doesn't work

**Review by powerje — CHANGES_REQUESTED** [review]

**Review by powerje — APPROVED** [review]

File: Fabulous.xcodeproj/project.pbxproj (hunk 2 above)
Comment (ThasianX) [reply Fabulous.xcodeproj/project.pbxproj]: Agreed. Was thinking more of appkit when i disabled mac

File: .gitignore (hunk 1 above)
Comment (ThasianX) [reply .gitignore:24]: Eh. Not using fastlane anyway

**Pull request merged**